lib/inc/internal/sources/dmi_source.hpp

```diff
@@ -52,6 +57,11 @@ namespace whereami { namespace sources {
         dmi_base() {}
         virtual ~dmi_base() {}
         /**
+         * Retrieve the BIOS address
+         * @return The BIOS address
+         */
+        std::string bios_address() const;
```

Comment:
Is this always available in the data from dmidecode, but just not always needed/useful for what we're trying to do? Or only under VMWare?